etcdserver/api/membership/cluster.go

```diff
@@ -776,3 +785,9 @@ func (c *RaftCluster) VotingMemberIDs() []types.ID {
 	sort.Sort(types.IDSlice(ids))
 	return ids
 }
+
+func (c *RaftCluster) IsAddedToCluster() bool {
```

Comment:
Let's add godoc for this function since it's exported even if it's duplicated from `EtcdServer.IsAddedToCluster()` below.

Reply:
`IsLocalNodeMember`?

Reply:
The trick is that even if `m.localID` is in `c.members`, it does not mean it was added by `AddMember` call. This flag is set to true when the local node is added (or overwrite, if it already exists) to `c.members` via 'AddMember'.